viewer/src/components/MoveButtonTooltip.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <b-tooltip :target="target" :title="title" triggers="hover" :variant="variant"> </b-tooltip>
```

Comment:
the title is html, but I didn't get that to work. This is what should work according to the docs:

```
  <b-tooltip :target="target" triggers="hover" :variant="variant"> 
    <div v-html="title"/>
  </b-tooltip>
```